sdk/python/feast/feature_store.py

```diff
@@ -998,6 +999,48 @@ def serve(self, port: int) -> None:
 
         feature_server.start_server(self, port)
 
+    @log_exceptions_and_usage
+    def upload_docker_image(self) -> None:
+        """Upload the docker image for the feature consumption server to the cloud."""
+
+        # TODO: add error checking and avoid hardcoding the region
+        repository_name = "feast-python-server-test"
+        feature_server_type = (
+            self.config.feature_server.type if self.config.feature_server else None
+        )
+        if feature_server_type == "aws_lambda":
+            import base64
+
+            import boto3
+            import docker
```

Comment:
You need to add this to setup.py. Also need to handle an error where docker binary isn't installed (I assume pip installing docker doesn't actually install the docker daemon)